### PR TITLE
Create CAN-RGX Operations Procedures

### PR DESCRIPTION
**Pull request by akmorrison**
Based off the procedures that Riley posted in #payload, just converted
to our sexy LaTeX template.

### CODE REVIEW

**Comment by kylemtam** [comment]
Errors to note:
N2 - line 1 "theremaining"
N2 - line 14 "holderand"
In Flight Operations - Level Flight Setup - first paragraph - "which cells" repeated
Reduced Gravity and Emergency Operations:
- line 4 - request to add "the" before "flight toolbox"
- line 10 - "someother"

File: can_rgx.tex [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+\textbf{End of Pre-Flight Setup}
+% end of pre-flight -- }}}
+
+% beginnning of in-flight checklist -- {{{
+\newpage{}
+\section{In-Flight Operations}
+% beginning of level flight setup -- {{{
+\subsection{Level Flight Setup}
+The following will be performed during every level flight section between
+parabolas.  A detailed list of which cells and which cells and magnet positions
```
Comment (kylemtam) [inline can_rgx.tex]: "which cells and" repeated

File: can_rgx.tex [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+The following will be performed for every reduced gravity section of flight.
+Exact details about the current configuration can be found in the table above.
+
+\begin{checklist}
+
+	\item The experiment will be left to run on its own, as it is fully passive. During operation, Mission Specialists are to be watching for anything that goes wrong or needs attention. This section is intended for use in case of mishaps. 
+	\item Ferrofluid Spill/leak:
+    \begin{checklist}
+		\item The leaking cell will be identified, removed from the test site and wrapped in absorbent cloth.
+		\item Cell and cloth will be placed in a large Ziploc bag, the bag will be sealed and placed in flight toolbox.
```
Comment (kylemtam) [inline can_rgx.tex]: request to say "in **the** flight toolbox"

File: can_rgx.tex [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+		\item If fluid has contaminated any parts of the experiment, it will be cleaned up with absorbent cloth and water (if necessary).
+		\item Leaking cell will be excluded from operations and the experiment may continue, provided there is no extensive damage to the system. 
+    \end{checklist}
+	\item Power Loss:
+    \begin{checklist}
+        \item Experiment will continue without charging to phone camera.
+    \end{checklist}
+	\item Camera Failure:
+    \begin{checklist}
+		\item Should the camera stop recording, or be incapacitated in someother way, it will be promptly removed from the experiment.
```
Comment (kylemtam) [inline can_rgx.tex]: someother

File: can_rgx.tex [hunk 4]
```diff
@@ -0,0 +1,248 @@
...
+    \end{checklist}
+\end{checklist}
+\textbf{End of Experiment Assembly}
+% end assembly -- }}}
+
+% beginnning of pre-flight setup checklist -- {{{
+\newpage{}
+\section{[N2] Pre-Flight Setup}
+\begin{checklist}
+	\item One at a time, theremaining valid test cells are to be filled with their designated amounts of ferrofluid using a syringe and pipette.
```
[the remark this answers is not in the document]
Comment (akmorrison) [reply can_rgx.tex]: fixed

File: can_rgx.tex [hunk 5]
```diff
@@ -0,0 +1,248 @@
...
+	\item After each fill, the test cell in question will be plugged:
+    \begin{checklist}
+		\item A shortened rubber stopper will be lightly greased with Vaseline, using a Q-tip.
+		\item The greased stopper will be pushed and then twisted into the fill port of the cell in question.
+		\item A thin bead of hot glue will be applied around the base of the stopper.
+    \end{checklist}
+	\item Each cell will be examined closely for any fluid leaks. Any leaking cells will be emptied and removed from the experiment.
+	\item Any changes necessary to cell numbering due to excluded test cells will be made now at the discretion of the Mission Specialists. 
+	\item When all cells are filled, they are to be loaded into their holding boxes and secured with the Velcro strap.
+	\item The camera phone will be secured in its holderand plugged into the pelican case for charging.
```
[the remark this answers is not in the document]
Comment (akmorrison) [reply can_rgx.tex]: fixed

File: can_rgx.tex (hunk 1 above)
Comment (akmorrison) [reply can_rgx.tex]: fixed

File: can_rgx.tex (hunk 2 above)
Comment (akmorrison) [reply can_rgx.tex]: fixed

File: can_rgx.tex (hunk 3 above)
Comment (akmorrison) [reply can_rgx.tex]: fixed

File: can_rgx.tex [hunk 6]
```diff
@@ -0,0 +1,248 @@
...
+% the current cost of such a purchase. If you do so, then use it                %
+% however you want, and I claim in no way that it compiles properly,            %
+% and I take no responsibilty for any loss you incurr from using                %
+% this template, and make no promises to support you in any technical           %
+% capacity for sustained use.                                                   %
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\input{common/standard_preamble}
+\usepackage{pdfpages}
+
+\title{
```
[the remark this answers is not in the document]
Comment (akmorrison) [reply can_rgx.tex]: this is actually done on purpose, the template is meant to be three-hole punched and put in a binder, at which point the logo is centered.

File: can_rgx.tex [hunk 7]
```diff
@@ -0,0 +1,248 @@
...
+    \item \textbf{N2} \textit{Pre-Flight Setup} comprised of all steps to be completed immediately before boarding the Falcon.
+    \item \textbf{N3} \textit{In-Flight Operations} comprised of setup, experiment procedures, and contingency plans to be executed aboard the Falcon.
+\end{itemize}
+% end contents subsection -- }}}
+
+% begin personnel required section -- {{{
+\subsection{Personnel Required}
+There are 4 people required for us performing at CAN-RGX.
+\begin{checklist}
+    \item \commanderfull{} is responsible for organizing and overseeing operations and crew, as well as liasing with SEDS and NRC staff. Primary operator for setup and experiment procedures.
```
[the remark this answers is not in the document]
Comment (akmorrison) [reply can_rgx.tex]: fixed

File: can_rgx.tex [hunk 8]
```diff
@@ -0,0 +1,248 @@
...
+        \end{checklist}
+        \item Plugged cells will be placed vertically on the shop towel and left for at least 30 minutes, longer if schedule allows.  Procedure shall continue while cells are sitting.
+    \end{checklist}
+    \item Connect camera remote to cell phone, and plug phone charger into Pelican Case.
+    \item Secure camera into phone holder, and plug into charger.
+    \begin{checklist}
+        \item Ensure phone is charging properly.
+        \item Use remote to start video on the camera.
+        \item Close case lid and use light remote to turn on interior lights to 80\% brightness. 
+        \item Wait several seconds, then turn off camera, light and open the Pelican Case.
```
[the remark this answers is not in the document]
Comment (akmorrison) [reply can_rgx.tex]: fixed

File: can_rgx.tex [hunk 9]
```diff
@@ -0,0 +1,248 @@
...
+    \item \textbf{N3} \textit{In-Flight Operations} comprised of setup, experiment procedures, and contingency plans to be executed aboard the Falcon.
+\end{itemize}
+% end contents subsection -- }}}
+
+% begin personnel required section -- {{{
+\subsection{Personnel Required}
+There are 4 people required for us performing at CAN-RGX.
+\begin{checklist}
+    \item \commanderfull{} is responsible for organizing and overseeing operations and crew, as well as liasing with SEDS and NRC staff. Primary operator for setup and experiment procedures.
+    \item \payloadfull{} is responsible for assisting the \commander{} with experiment setup and operation. Both he and the \commander{} will be flying in reduced gravity aboard the Falcon.
```
[the remark this answers is not in the document]
Comment (akmorrison) [reply can_rgx.tex]: Replaced with "Payload Specialist" instead of "he"

File: can_rgx.tex [hunk 10]
```diff
@@ -0,0 +1,248 @@
...
+    \item \mspecsecfull       \tabto{25em}\rule{10em}{0.4pt}\hspace{5em}\rule{10em}{0.4pt}
+\end{checklist}
+\setcounter{checklistnum}{0}
+% end sign off -- }}}
+
+% begin assembly checklist-- {{{
+\newpage
+\section{[N1] Experiment Assembly}
+\begin{checklist}
+    \item Using 3/16'' and 4mm Allen keys for 1/4-20 and M5 screws (respectively), check tightness of all fasteners.
```
[the remark this answers is not in the document]
Comment (akmorrison) [reply can_rgx.tex]: done

**Comment by akmorrison** [comment]
@kylemtam @tteresa can y'alls approve and merge this?

**Pull request merged**